src/poetry/console/commands/check.py

```diff
@@ -73,6 +79,22 @@ def validate_classifiers(
 
         return errors, warnings
 
+    def validate_readme(
+        self, readme: str | list[str], poetry_file: Path
+    ) -> tuple[list[str], list[str]]:
+        """Check existence of referenced readme files"""
+
+        errors = []
+        warnings: list[str] = []
```

Comment:
`warnings` is not used. Why it is created?

Reply:
Probably, so that it has the same return values as `validate_classifiers()`. On the other side, we could also omit warnings as long as we don't need it. No strong feelings about it. 🤷

Reply:
Yes, as I mentioned above, there're 2 reasons:

- to be consistent with `validate_classifiers()`.
- for future extensibility. Say, Python 3.14 introduces a cheap way to detect the real cases of files, then we can raise warnings for inconsistent cases.

I have no strong idea either, should I remove it?

Reply:
Maybe, just remove it for now and put a `_` in front of the validate-methods (`_validate_classifiers`, `_validate_readme`) to make clear they are internal and the interface might be changed in future.

If we introduce the third, fourth or fifth validate method, we should think about a common interface (not only return values, but also parameters) and calling the methods in a loop anyway.

Reply:
Make sense. I did try to make it into a loop (in January), but I found it hard to unify parameters.